src/main/java/org/openelisglobal/workplan/controller/WorkplanByTestSectionRestController.java

```diff
@@ -0,0 +1,277 @@
+import org.openelisglobal.analysis.service.AnalysisService;
+import org.openelisglobal.analysis.valueholder.Analysis;
+import org.openelisglobal.common.constants.Constants;
+import org.openelisglobal.common.formfields.FormFields;
+import org.openelisglobal.common.formfields.FormFields.Field;
+import org.openelisglobal.common.services.DisplayListService;
+import org.openelisglobal.common.services.QAService;
+import org.openelisglobal.common.services.DisplayListService.ListType;
+import org.openelisglobal.common.services.QAService.QAObservationType;
+import org.openelisglobal.common.util.ConfigurationProperties;
+import org.openelisglobal.common.util.IdValuePair;
+import org.openelisglobal.common.util.ConfigurationProperties.Property;
+import org.openelisglobal.observationhistory.service.ObservationHistoryService;
+import org.openelisglobal.observationhistory.service.ObservationHistoryServiceImpl.ObservationType;
+import org.openelisglobal.patient.service.PatientService;
+import org.openelisglobal.person.service.PersonService;
+import org.openelisglobal.result.action.util.ResultsLoadUtility;
+import org.openelisglobal.role.service.RoleService;
+import org.openelisglobal.sample.service.SampleService;
+import org.openelisglobal.sample.valueholder.Sample;
+import org.openelisglobal.samplehuman.service.SampleHumanService;
+import org.openelisglobal.sampleqaevent.service.SampleQaEventService;
+import org.openelisglobal.sampleqaevent.valueholder.SampleQaEvent;
+import org.openelisglobal.search.service.SearchResultsService;
+import org.openelisglobal.spring.util.SpringContext;
+import org.openelisglobal.systemuser.service.UserService;
+import org.openelisglobal.test.beanItems.TestResultItem;
+import org.openelisglobal.test.service.TestSectionService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.ResponseBody;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/rest/")
+public class WorkplanByTestSectionRestController extends WorkplanRestController {
+
+	@Autowired
+	SampleService sampleService;
+	@Autowired
+	PatientService patientService;
+	@Autowired
+	PersonService personService;
+	@Autowired
+	ObservationHistoryService observationHistoryService;
+	@Autowired
+	SampleHumanService sampleHumanService;
+	@Autowired
+	SearchResultsService searchResultsService;
+	@Autowired
+	private AnalysisService analysisService;
+	@Autowired
+	private SampleQaEventService sampleQaEventService;
+	
+	@Autowired
+	private RoleService roleService;
+	@Autowired
+	private UserService userService;
+
+
+	@GetMapping(value = "test-sections", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
can you put thiese common Lists under [common](https://github.com/I-TECH-UW/OpenELIS-Global-2/blob/2.8/src/main/java/org/openelisglobal/common/rest/DisplayListController.java)